examples/web-ui/material/pages/Design/designHome.smash

```diff
@@ -0,0 +1,17 @@
+* Given I am on design page
+    Navigate to design page
+        On design page
```

Comment:
`Navigate to design page` already calls `On design page`. You should only call that here or in that function.